### PR TITLE
feat: add CollectorServiceAccountsSection read case markups

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue [hunk 1]
```diff
@@ -12,17 +12,108 @@
...
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+interface SecretModel {
+    secret_id: string;
+    provider: string;
+    service_account_id: string;
+    project_id: string;
+    created_at: string;
+}
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue:45]: suggestions: We were managing types in a separate file, is there a specific guideline for that?

**Review by skdud4659 — APPROVED** [review]
LGTM~!!! I left some suggestions!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue:45]: Oh this is temporary type declaration! I forgot to left a comment! Sorry ~

**Review by yuda110 — APPROVED** [review]
LGTM!!! Sorry for the delay!

**Pull request merged**